tests/test_pytest_cov.py

```diff
@@ -889,6 +889,38 @@ def test_funcarg_not_active(testdir):
     assert result.ret == 0
 
 
+def test_multiprocessing_pool(testdir):
+    py.test.importorskip('multiprocessing.util')
+
+    script = testdir.makepyfile('''
+import multiprocessing
+
+def target_fn(a):
+    return a + 1
+
+def test_run_target():
+    for i in range(100):
+        with multiprocessing.Pool(10) as p:
```

Comment:
Why this test doesn't use the recommended way https://github.com/pytest-dev/pytest-cov/pull/265/files#diff-c1d4de3cfc01566955163b584f0efcc0R29 ?

Reply:
So I added support for the old way while fixing the regressions. Need to update the docs again. The new way is still the best way for the pytest-cov<=2.6.1 releases.